packages/interface-transport/src/index.ts

```diff
@@ -78,16 +79,22 @@ export interface UpgraderEvents {
   'connectionEnd': CustomEvent<Connection>
 }
 
+export interface UpgraderOptions {
+  skipEncryption?: boolean
+  skipProtection?: boolean
+  muxerFactory?: StreamMuxerFactory | undefined
```

Comment:
If the field is specified it should have the right type and not be undefined.

```suggestion
  muxerFactory?: StreamMuxerFactory
```

Reply:
Sorry for the final nit - I'd merge this but the PR is set to not allow pushing extra commits.

Reply:
done